authentik/blueprints/apps.py

```diff
@@ -42,6 +44,33 @@ def reconcile(self) -> None:
             except (DatabaseError, ProgrammingError, InternalError) as exc:
                 self._logger.debug("Failed to run reconcile", name=name, exc=exc)
 
+    def reconcile_tenant(self) -> None:
+        """reconcile ourselves for tenanted methods"""
+        from authentik.tenants.models import Tenant
+
+        try:
+            tenants = list(Tenant.objects.filter(ready=True))
+        except (DatabaseError, ProgrammingError, InternalError) as exc:
+            self._logger.debug("Failed to get tenants to run reconcile", exc=exc)
+            return
+        for tenant in tenants:
```

Comment:
I think the loop could be in the `try` block, which would get rid of the variable and the `list` call

Reply:
The idea of having this try/catch outside of the loop was to specifically catch errors related to the tenant query, and not to whatever _reconcile is doing.

Reply:
fair enough